client/tomcatconf/commands.properties.in

```diff
@@ -485,6 +485,10 @@ updateVPCOffering=1
 deleteVPCOffering=1
 listVPCOfferings=15
 
+#### VPC OSPF config commands
+vpcOSPFConfig=15
+vpcOSPFConfigUpdate=1
```

Comment:
@rhtyd is it necessary to do this change for 4.9 or can be done later. I am asking this as I checked some other APIs and did not see the annotation.